tsconfig.base.json

```diff
@@ -25,6 +25,7 @@
         "libs/agora/gene-comparison-tool/src/index.ts"
       ],
       "@sagebionetworks/agora/gene-details": ["libs/agora/genes/src/index.ts"],
+      "@sagebionetworks/agora/gene-similar": ["libs/agora/genes/src/index.ts"],
```

Comment:
This line may be referencing the incorrect index file (currently the same as `@sagebionetworks/agora/gene-details`).

Reply:
Same comment actually for `@sagebionetworks/agora/gene-details`, which references the same index file as `@sagebionetworks/agora/genes`.

Reply:
Good catch.  I restructured this from the original and ended up with duplicates.  I've deleted the gene-details and gene-similar entries and updated any references to just use @sagebionetworks/agora/genes.

Reply:
This has not been fixed yet.